packages/rpc-core/src/params-patcher-options-object-position-config.ts

```diff
@@ -0,0 +1,51 @@
+export const OPTIONS_OBJECT_POSITION_BY_METHOD: Record<string, number> = {
+    accountNotifications: 1,
+    blockNotifications: 1,
+    getAccountInfo: 1,
+    getBalance: 1,
+    getBlock: 1,
+    getBlockCommitment: 1,
```

Comment:
This one doesn't seem to have a `commitment` parameter [on the documentation](https://docs.solana.com/api/http#getblockcommitment). It could be under-documented though.

Reply:
OMG amazing. This is exactly the kind of review I needed.

![giphy.gif](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lE1zD7Jpg6mWNv3djNKZ/cfce0580-333c-4e42-aaa3-e8f09bc48a7c.gif)